### DISCUSSION TITLE
What's a good 0-10v pressure transducer for filter pressure?

### DISCUSSION THREAD

**Discussion by jamestalmage** [tagyoureit/nodejs-poolController#846] [Nixie]
Amazon's turning up lots of options that are either expensive, no reviews, or 4-20ma (I've only got the MEGA-BAS, so I'd like to stick with 0-10v if I could)

**Comment by rstrouse** [comment]
You should be able to wire up any 5v transducer to the 0-10v connectors on the BAS.  Not sure how it is wired though.

**Reply by tagyoureit** [reply]
I previously got mine on eBay.  They have 5v 100ps pressure transducers for <$25.  The 10v transducers are much more expensive.
